test/test-coordinator.js

```diff
@@ -165,11 +165,27 @@ TestCoordinator.prototype.startSUT = function startSUT() {
     var newProc;
     var hostsFileArg = util.format('--hosts=%s', this.hostsFile);
     var listenArg = util.format('--listen=%s', this.sutHostPort);
+    var suspectPeriodArg = util.format('--suspect=5')
+    var faultyPeriodArg = util.format('--faulty=5')
+    var tombstonePeriodArg = util.format('--tombstone=5')
     console.log(this.sutProgram, listenArg, hostsFileArg);
     if (this.sutInterpreter) {
```

Comment:
If we add more arguments it might be better to refactor this bit a bit. Something like:
```lang=javascript
var program;
var arguments = [];
if (this.sutInterpreter) {
    program = this.sutInterpreter;
    arguments.push(this.sutProgram);
} else {
    program = this.sutProgram;
}
var arguments = [];
arguments.push(util.format('--hosts=%s', this.hostsFile));
arguments.push(util.format('--listen=%s', this.sutHostPort);
arugments.push(util.format('--suspect=5'));
// etc. other arguments.

newProc = childProc.spawn(program, arguments);
```